src/js/videojs-dash.js

```diff
@@ -70,6 +70,71 @@ class Html5DashJS {
     // element to bind to.
     this.mediaPlayer_.initialize();
 
+    // Retrigger a dash.js-specific error event as a generic HTML5 one
+    // See src/streaming/utils/ErrorHandler.js in dash.js code
+    // Handled with error (playback is stopped):
+    // - capabilityError
+    // - downloadError
+    // - manifestError
+    // - mediaSourceError
+    // - mediaKeySessionError
+    // Not handled:
+    // - timedTextError (video can still play)
+    // - mediaKeyMessageError (only fires under 'might not work' circumstances)
+    this.retriggerError_ = (event) => {
+      // No support for MSE
+      if ((event.error === 'capability' && event.event === 'mediasource') ||
+        // Manifest type not supported
+        (event.error === 'manifestError' && event.event.id === 'createParser') ||
+        // Codec(s) not supported
+        (event.error === 'manifestError' && event.event.id === 'codec') ||
+        // No streams available to stream
+        (event.error === 'manifestError' && event.event.id === 'nostreams') ||
+        // not sure about this one
```

Comment:
Looks like it catches various possible errors when creating stream objects from manifest data? Though I only briefly looked into it.

Reply:
Yeah, I'm just not sure what the `composed` is all about.